### DISCUSSION TITLE
Transfer Bookmarks to new mac

### DISCUSSION THREAD

**Discussion by csuebws** [iterate-ch/cyberduck#15302] [Q&A] [answered]
How do I transfer my bookmarks to a new mac? Thanks

**Comment by ylangisc** [comment, marked as answer]
Just transfer the folder `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks` to your new computer.

**Reply by dkocher** [reply]
Documented in [Preferences and Application Support Files Location](https://docs.cyberduck.io/cyberduck/faq/#preferences-and-application-support-files-location).

**Comment by csuebws** [comment]
Thanks! It worked!
